### PR TITLE
Core: Added conversion between OptionProperty and IntProperty.

### PR DESCRIPTION
**Pull request by Alboin**
**What evineroment has the code been compiled and tested on?** 
- Windows 10
- Visual Studio 15.7.6
- QT Community 4.6.1
- CMake 3.11.2
- Python 3.6 (32-bit)

### CODE REVIEW

File: src/core/common/inviwocore.cpp [hunk 1]
```diff
@@ -358,9 +371,17 @@ InviwoCore::InviwoCore(InviwoApplication* app)
 
     using OptionTypes = std::tuple<unsigned int, int, size_t, float, double, std::string>;
     util::for_each_type<OptionTypes>{}(OptionStringConverterRegFunctor{}, registerPC);
+    using OptionTypes = std::tuple<unsigned int, int, size_t, float, double, std::string>;
```
Comment (martinfalk) [inline src/core/common/inviwocore.cpp]: why duplicating the `OptionTypes`?

File: src/core/common/inviwocore.cpp [hunk 2]
```diff
@@ -358,9 +371,17 @@ InviwoCore::InviwoCore(InviwoApplication* app)
...
+    util::for_each_type<OptionTypes>{}(OptionIntConverterRegFunctor{}, registerPC);
+    using OptionTypes = std::tuple<unsigned int, int, size_t, float, double, std::string>;
+    util::for_each_type<OptionTypes>{}(IntOptionConverterRegFunctor{}, registerPC);
 
     using OptionEnumTypes = std::tuple<OptionRegEnumInt, OptionRegEnumUInt>;
     util::for_each_type<OptionEnumTypes>{}(OptionStringConverterRegFunctor{}, registerPC);
+    using OptionEnumTypes = std::tuple<OptionRegEnumInt, OptionRegEnumUInt>;
```
Comment (martinfalk) [inline src/core/common/inviwocore.cpp]: should not be necessary to repeat `OptionEnumTypes`

**Comment by martinfalk** [comment]
This is quite handy!

File: src/core/common/inviwocore.cpp (hunk 1 above)
Comment (Alboin) [reply src/core/common/inviwocore.cpp]: Good question!

File: src/core/common/inviwocore.cpp (hunk 2 above)
Comment (Alboin) [reply src/core/common/inviwocore.cpp]: Nope!

**Review by martinfalk — APPROVED** [review]
looks good.

**Review by petersteneteg — APPROVED** [review]

**Pull request merged**